src/app/authentication/authentication.component.ts

```diff
@@ -10,6 +10,7 @@ import { LoginAuthService } from "src/app/login-auth.service";
   styleUrls: ["./authentication.component.scss"]
 })
 export class AuthenticationComponent implements OnInit {
+  isTrue: boolean = false;
```

Comment:
isTrue - not telling me anything.
You using this flag to show an error message when authentication is failed. 
it should be called something like **authonticationFailed**